Meadowlark-js/services/meadowlark-fastify/src/Cluster.ts

```diff
@@ -17,17 +17,18 @@ export class ClusterService {
   /** Run in multi-threaded mode */
   run() {
     if (cluster.isPrimary) {
-      this.master();
+      ClusterService.primary();
     } else {
-      // eslint-disable-next-line @typescript-eslint/no-floating-promises
       this.worker();
     }
   }
 
-  // eslint-disable-next-line class-methods-use-this
-  master() {
-    Logger.debug(`Total Number of Cores: ${CPUS}`, null);
-    Logger.debug(`Master ${process.pid} is running`, null);
+  static primary(): void {
```

Comment:
Changed name to avoid the term "master", and made it static since it doesn't reference `this`.

Reply:
This is fine but if it were me, I'd just take the function out of the class.